src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -449,13 +459,17 @@ class DocAnnotator extends Annotator {
             this.annotatedElement.removeEventListener('dblclick', this.highlightMouseupHandler);
             this.annotatedElement.removeEventListener('mousedown', this.highlightMousedownHandler);
             this.annotatedElement.removeEventListener('contextmenu', this.highlightMousedownHandler);
-            this.annotatedElement.removeEventListener('mousemove', this.getHighlightMouseMoveHandler());
-
-            if (this.highlightThrottleHandle) {
-                cancelAnimationFrame(this.highlightThrottleHandle);
-                this.highlightThrottleHandle = null;
+            if (this.hasTouch && this.isMobile) {
+                document.removeEventListener('selectionchange', this.onSelectionChange);
+            } else {
+                this.annotatedElement.removeEventListener('mousemove', this.getHighlightMouseMoveHandler());
             }
         }
+
+        if (this.highlightThrottleHandle) {
```

Comment:
That's no used for annotating. That's used for hovering over highlights. Not sure what our conclusion was about cascading browser events, in the case you're using a mouse on a tablet, so I'll make a card to investigate removing this on mobile.